test.sh

```diff
@@ -75,7 +76,7 @@ function setup_osbs() {
   fi
 
   # Pip install packages for atomic-reactor
-  $RUN "${PIP_INST[@]}" -r requirements.txt
+  $RUN "${PIP_INST[@]}" -r requirements.txt --ignore-installed
```

Comment:
Why `--ignore-installed`? If it's necessary, could you add a comment?

Reply:
added comment

Reply:
> it was added due to PyGoObject conflicting with the existing PyGoObject on fedora. I'll add a comment.

When/how did this happen? "It Used To Work"™️